src/Internal/Serialization/PlutusData.purs

```diff
@@ -23,38 +23,39 @@ import Ctl.Internal.Types.BigNum (fromBigInt) as BigNum
 import Ctl.Internal.Types.ByteArray (ByteArray)
 import Ctl.Internal.Types.PlutusData as T
 import Data.BigInt as BigInt
-import Data.Maybe (Maybe)
-import Data.Traversable (for, traverse)
+import Data.Maybe (Maybe(Just), fromJust)
 import Data.Tuple (Tuple, fst, snd)
 import Data.Tuple.Nested (type (/\), (/\))
+import Partial.Unsafe (unsafePartial)
 
-convertPlutusData :: T.PlutusData -> Maybe PlutusData
-convertPlutusData = case _ of
+convertPlutusData :: T.PlutusData -> PlutusData
+-- Unsafe fromJust here is correct, because we cover every PlutusData
+-- constructor, and Just will be returned by one of functions
```

Comment:
This isn't true, convertConstr can fail due to BigInt -> BigNum conversion failure

Reply:
Hmmm, no reason to use BigInt then, right? Let's switch @uhbif19

Reply:
@klntsky Switched. But there is still `convertPlutusInteger` case. There is conversion from one BigInt to another BigInt which leads to Maybe. But I do not understand what is the difference between BigInts so do not know if it safe to remove Maybe.

Reply:
Looked into it, made a change request